tests/integration/test_author_api.py

```diff
@@ -42,35 +42,45 @@ def test_api_authors_root(app):
 def test_api_authors_citations(app):
     schema = {
         'items': {
+            'type': 'object',
             'properties': {
                 'citee': {
+                    'type': 'object',
                     'properties': {
                         'record': {
+                            'type': 'object',
                             'properties': {'$ref': {'type': 'string'}},
-                            'type': 'object'
-                        }
-                    },
-                    'type': 'object'
+                        },
+                        'id': {'type': 'integer'}
+                    }
                 },
-                'citer': {
-                    'properties': {
-                        'record': {
-                            'properties': {'$ref': {'type': 'string'}},
-                            'type': 'object'
+                'citers': {
+                    'type': 'array',
+                    'items': {
+                        'type': 'object',
+                        'properties': {
+                            'date': {'type': 'string'},
+                            'self_citation': {'type': 'boolean'},
+                            'citer': {
+                                'type': 'object',
+                                'properties': {
+                                    'record': {
+                                        'type': 'object',
+                                        'properties': {
+                                            '$ref': {'type': 'string'}
+                                        }
+                                    },
+                                    'id': {'type': 'integer'}
+                                }
+                            },
+                            'published_paper': {
+                                'type': 'boolean'
+                            }
                         }
-                    },
-                    'type': 'object'
-                },
-                'date': {
-                    'format': 'date',
-                    'type': 'string'
-                },
-                'published_paper': {'type': 'boolean'},
-                'self_citation': {'type': 'boolean'}
-            },
-            'type': 'object'
-        },
-        'type': 'array'
+                    }
+                }
+            }
+        }
     }
```

Comment:
I'm having some trouble picturing what does a response look like in this schema. Can you write an example in https://github.com/inspirehep/inspire-next/issues/1416 ?